mention about monthly meeting in NCA responsibilities (#31764)
kubernetes/website: content/en/docs/contribute/advanced.md

- Mentoring new contributors through their first few PRs to the docs repo.
- Helping new contributors create the more complex PRs they need to become Kubernetes members.
- [Sponsoring contributors](/docs/contribute/advanced/#sponsor-a-new-contributor) on their path to becoming Kubernetes members.

> - Hosting a monthly meeting with an existing SIG Docs contributor for new contributors.

Reviewer: It's a little unclear to me - does the NCA organize a meeting that an existing contributor attends, so that new contributors can ask questions?

Reviewer 2: ```suggestion
- Hosting a monthly meeting to help and mentor new contributors.
```
How's this change? The bullets above state the NCA mentors and helps new contributor and the monthly meeting can be a virtual venue for this.

Reviewer: +1 for Rey's suggestion :)

Author: Ah yes thank you that's much clearer! 😄
